upup/pkg/fi/cloudup/awstasks/subnet.go

```diff
@@ -178,6 +179,19 @@ func (_ *Subnet) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Subnet) error {
 		}
 
 		e.ID = response.Subnet.SubnetId
+
+	}
+
+	// Tagging according to upstream so that internal and external ELBs
+	// are deterministically created where they are supposed to be.
+	// https://github.com/kubernetes/kops/issues/2011
+	glog.V(4).Infoln("Found a subnet of type: %s and tagging it for proper ELB placement", e.Type)
+	if e.Type == kops.SubnetTypePrivate {
+		e.Tags["kubernetes.io/role/internal-elb"] = ""
```

Comment:
I think we do tagname = 1 elsewhere.

Reply:
dont we need another tag as well. Checking

Reply:
Do we also need the tags from here https://github.com/kubernetes/kubernetes/pull/41695 shared and owned? I am not certain, but since you are messing with tags anyways

Reply:
Those tags are generated in https://github.com/kubernetes/kops/blob/master/pkg/resources/aws.go

I'll change values to 1. It also occurred to me that folks who use TF and cloud formation probably also want this, so refactoring a bit.